pkg/controller/risingwave_scale_view_controller.go

```diff
@@ -106,6 +109,11 @@ func (c *RisingWaveScaleViewController) Reconcile(ctx context.Context, request r
 }
 
 func (c *RisingWaveScaleViewController) SetupWithManager(mgr ctrl.Manager) error {
+	gvk, err := apiutil.GVKForObject(&risingwavev1alpha1.RisingWaveScaleView{}, c.Client.Scheme())
```

Comment:
`Reconcile` function is not covered. I think we should do this is a different PR. Could you please open an issue for that? (see `go tool cover -html=cover.out`)